### ISSUE TITLE
Screen blanks out on Scanning/Building of rocket [solved]

### ISSUE THREAD

**Issue by gdarai** [zmaster587/AdvancedRocketry#887]
[1.10.2] This is some visual bug.
When I am looking on the Scanning/Building animation thw whole world disapears (visually). And it reapears when the animation is finished.

I thought that it has something to do with the new release or the AdvancedAnimations setting... but to. The previous version with AdvancedFX off does it as well.

At least I can reduce the time by reducing the Scanning/Testing time but that is the opposite of what I wanted to have.

**Comment by gdarai** [comment]
Fixed... it was that BetterFPS. Is there some place to write such MODS cross problems?

**Comment by zmaster587** [comment]
there's no central cross platform mod thing that that I'm aware of, though due to the nature of gameplay and bugs in advanced rocketry unfortunately this issue has been put on back burner for quite some time.

**Comment by andreivasiliu** [comment]
I've stumbled upon this bug while playing Forever Stranded, just looking at a completed Space Elevator or at a rocket/station scan/build in progress made my whole screen turn grayish. The color of the screen changed a bit depending on the direction I was looking.

When scanning/building, looking away from the launch pad made me able to see again, looking back at it made it back gray. With the space elevator, the only way to regain vision was to break it.

I managed to fix this by going to BetterFPS's options, turning Fog from Fast to Fancy, and restarting the game.
